### ISSUE TITLE
Bump dependencies and new release?

### ISSUE THREAD

**Issue by brevilo** [ma1uta/jmsdk#17]
Hi! 

Development seems to be paused right now. Could you please consider bumping the dependencies again and push another release so that the latest version is available via maven?

Thanks and keep it up!

**Comment by brevilo** [comment]
Ideally containing the fix for #18 😏

**Comment by brevilo** [comment]
Speaking of dependencies, even while mostly unrelated to this issue...

Here's a little FYI for everyone trying to deploy this in an OSGi environment: you need at least version [2.0.1-RC1](https://github.com/eclipse/microprofile-rest-client/releases/tag/2.0.1-RC1) of the `microprofile-rest-client-api` or the runtime deployment won't work (due to unresolvable dependencies on CDI and MircoProfile Config).

Hope this helps

**Comment by ma1uta** [comment]
I have plan to bump dependencies, yes. I think I can do this in a few weeks, sorry for delay.

**Comment by brevilo** [comment]
No worries, I have another PR coming (needed for OSGi compatibility). Hope to get it done later today.

**Comment by brevilo** [comment]
Also, there might might be another bugfix required for `microprofile-rest-client-api` so holding off is actually a good thing right now. I'll keep you posted if and when things got resolved.

**Comment by brevilo** [comment]
> No worries, I have another PR coming (needed for OSGi compatibility). Hope to get it done later today.

There you go: #20

**Comment by blakehawkins** [comment]
I'm also interested in this for https://github.com/ma1uta/jmsdk/pull/11

**Comment by brevilo** [comment]
> Also, there might might be another bugfix required for `microprofile-rest-client-api` so holding off is actually a good thing right now. I'll keep you posted if and when things got resolved.

This needs some further work. So if you want to bump deps and release, please go ahead. Please remember to update the rest client as [described above](https://github.com/ma1uta/jmsdk/issues/17#issuecomment-811271928).

**Comment by ma1uta** [comment]
Done in https://github.com/ma1uta/jmsdk/commit/f3c8bed45a7b3deeff10f8532dcbb71376dedd1c I'm going to publish release.

**Comment by ma1uta** [comment]
I use microprofile 3.3: https://github.com/ma1uta/jmsdk/blob/master/jmsdk-bom/pom.xml#L58

**Comment by brevilo** [comment]
Ah, right. Well, in that case I need to maintain my own fork for the time being. Even the latest  `microprofile` (4.0.1) still provides only `microprofile-rest-client-api` 2.0, for obvious reasons.

I'll close this issue. I'll ask for a `microprofile` bump when they released a version that includes the rest API >= 2.0.1. That's probably going to take a while.

Thanks

**Comment by brevilo** [comment]
Ok, I'll reopen until `0.13.0` is actually released 😎 (please do)

**Comment by ma1uta** [comment]
I'm trying but stuck on error when building the master:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project jackson-support: Compilation failure
[ERROR] ~/dev/jmsdk/jackson-support/src/main/java/module-info.java:[31,52] module not found: com.fasterxml.jackson.jaxrs.json
```

I think I'll fix this.

**Comment by brevilo** [comment]
Yep, I've also been looking into this and couldn't find a solution.

OTOH, glad you updated MapStruct to >= 1.4.0 since they had an [issue](https://github.com/mapstruct/mapstruct/issues/2060) with Java 11 modules (`java.sql.Date` missing).

**Comment by brevilo** [comment]
About the Jackson issue: since things aren't getting fixed upstream, how about reverting the earlier [version bump](https://github.com/ma1uta/jmsdk/commit/f3c8bed45a7b3deeff10f8532dcbb71376dedd1c) from 2.12.3 back to 2.11.2? I think I could confirm [2.12.0-rc2](https://github.com/FasterXML/jackson-jaxrs-providers/issues/144#issuecomment-825209190) as being the culprit, so the latest in the 2.11 series, i.e. 2.11.4 should also be fine.

This way we could get things going again...

**Comment by ma1uta** [comment]
Ok, I have release 0.13.1 (jeon and jmsdk) with jackson 2.11.4, please see, does it works?

**Comment by brevilo** [comment]
Yep, works for me. Should we open a issue to remind ourselves about the pending upstream issue, or keep this one open?

**Comment by ma1uta** [comment]
As you wish :)
